api/v1alpha1/dnspolicy_types.go

```diff
@@ -125,6 +130,9 @@ type DNSPolicyStatus struct {
 
 	// +optional
 	RecordConditions map[string][]metav1.Condition `json:"recordConditions,omitempty"`
+	// TotalRecords records the total number of individual DNSRecords managed by this DNSPolicy
+	// +optional
+	TotalRecords int32 `json:"totalRecords,omitempty"`
```

Comment:
@mikenairn added this maybe not needed? Thought it was a useful thing and planned to use it elsewhere